src/python/WMComponent/JobCreator/JobCreatorPoller.py

```diff
@@ -109,6 +110,10 @@ def capResourceEstimates(jobGroups, constraints):
             j['estimatedJobTime'] = min(j['estimatedJobTime'], constraints['MaxWallTimeSecs'])
             j['estimatedDiskUsage'] = min(j['estimatedDiskUsage'], constraints['MaxRequestDiskKB'])
 
+            # finally, quantize those
+            j['estimatedJobTime'] = quantize(j['estimatedJobTime'], constraints['MinWallTimeSecs'])
```

Comment:
Is this enough?  I thought "MaxWallTimeMins" has to be quantized.
```
MaxWallTimeMins = WM
Core_ResizeJob ? ( EstimatedSingleCoreMins / RequestCpus + 15 ) : OriginalMaxWallTimeMins
```
I mean  due to RequestCpus and '+15', it won't have the same quantization you intended.
I am not sure where the 15 comes form if we want to quantize 100's, that should be just removed.

Reply:
I think it's fine. MaxWallTimeMins is built on top of `estimatedJobTime` and the most important is that all jobs reaching the plugin are quantized already. As you noted, it will vary according to the number of cpus, but at least it will always use the same discrete value range.

Reply:
Hmm, according to @juztas post it seem MaxWallTimeMins is used for matching  #7631
So shouldn't that be quantized? Although estimatedJobTime is quantized, won't it change granularity due to division below. (EstimatedSingleCoreMins / RequestCpus + 15) Also I am not sure + 15 is necessary if the quantization is bigger than that.

Reply:
The only way to get that quantized as well would be using some HTCondor rounding/quantization. AFAIK it has a different algorithm and it isn't very efficient.

I think quantizing the estimatedJobTime will decrease the number of auto-cluster by a lot already (in theory from 2880 to 48 combinations).

Maybe @bbockelm can share his thoughts in here...

Reply:
Note that `MaxWallTimeMins` is an expression that is evaluated on the fly, not a value.  You can't quantize a priori -- HTCondor has to re-evaluate it.

Also, in terms of clusters and matchmaking, the idle job are all identical until *after* matchmaking is done (when the real value of CPUs is filled in).  So we can effectively ignore all autoclusters that only contain running jobs.